packages/pages/src/components/image/image.tsx

```diff
@@ -1,14 +1,34 @@
 import * as React from "react";
-import { useRef, useState, useEffect } from "react";
-import { ImageProps } from "./types";
-
+import { useEffect, useRef, useState } from "react";
+import { ImageProps, ImageLayout } from "./types";
+
+/**
+ * A component that renders an image. Here is an example of using the component with simple and
```

Comment:
I'd rephrase this intro to:

Renders an image based from the Yext Knowledge Graph